include/SSVOpenHexagon/Core/HGStatus.hpp

```diff
@@ -4,12 +4,19 @@
 
 #pragma once
 
+#include <sys/time.h>
```

Comment:
I'm going to be a pain and ask you if you can change this to use `<chrono>`, which is the portable way of performing time-related operations in C++11 and above. `<sys/time.h>` is not portable.

https://en.cppreference.com/w/cpp/chrono/